plugins/guests/ubuntu/cap/change_host_name.rb

```diff
@@ -6,6 +6,23 @@ def self.change_host_name(machine, name)
           super
         end
 
+        def initialize(machine, new_hostname)
+          super
+          @logger = Log4r::Logger.new("vagrant::plugins::guest")
+        end
+
+        def get_current_hostname
+          return super unless systemd?
+
+          hostname = ""
+          sudo "hostnamectl --static status" do |type, data|
+            hostname = data.chomp if type == :stdout && hostname.empty?
+          end
+          @logger.debug("ubuntu get_current_hostname got hostname=#{hostname} by hostnamectl.")
```

Comment:
@chrisvire I removed them too. Thanks for a catch!
This time I checked no logging is in the result of `git diff master`.